lib/pact.ex

```diff
@@ -110,6 +120,21 @@ defmodule Pact do
     {:stop, :normal, :ok, container}
   end
 
+  defp create_fake_module_name(name, pid) do
+    process_id_as_string = IO.iodata_to_binary(:erlang.pid_to_list(pid))
```

Comment:
That's how you get the id of the pid. It's ripped from the `IO.inspect` for `PID`. You can't call pass a process to `to_string` either :(